### PR TITLE
Update the docs building process

### PR DESCRIPTION
**Pull request by djmitche**
This now requires some extra packages, one of which depends on Pillow
which requires libjpeg.

Also, the Makefile requires GNU make.

@tardyp does this look good?

### CODE REVIEW

File: jail-docs.yml [hunk 1]
```diff
@@ -36,7 +40,7 @@
     target_commands:
     # `command` module does not allow to set env variables, nor expands them,
     # so let's use make's variables.
-    - "make -C master/docs PATH:={{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin:$(PATH) VERSION=latest"
+    - "gmake -C master/docs PATH:={{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin:$(PATH) VERSION=latest"
```
Comment (seankelly) [inline jail-docs.yml]: Does `jpeg`, or another package, require that gmake be installed?

Comment (tardyp) [inline jail-docs.yml]: no, its an actual unrelated problem
The docs makefile has some gnuisms, which we saw in the metabb

Comment (seankelly) [inline jail-docs.yml]: Sorry, I meant will the `jpeg` package install the `gmake` package?

Comment (djmitche) [inline jail-docs.yml]: Oh, interesting, gmake was installed already on bslave1 so I assumed it would be here, too.  It's not!

**Comment by djmitche** [comment]
how's that?

**Comment by seankelly** [comment]
It's not in alphabetic order but I'll deal. I can't comment on the Python packages but the rest looks good to me.

**Comment by djmitche** [comment]
I'll sort it and push, thanks

**Pull request merged**